api/v1/utils/crypto.js

```diff
@@ -1,10 +1,13 @@
+/* jshint esversion: 6 */
+
 var Sha = require('jssha');
 
-var SHA_TYPE = "SHA-256";
-var SHA_INPUT_TYPE = "TEXT";
-var SHA_OUTPUT_TYPE = "HEX";
+const SHA_TYPE = "SHA-256";
+const SHA_INPUT_TYPE = "TEXT";
+const SHA_OUTPUT_TYPE = "HEX";
 
-var DEFAULT_PASSWORD_LENGTH = 33;
+const PASSWORD_RADIUS = 31;
```

Comment:
Do you meant to call it "radix" or "base" instead of radius? See [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString).